tests/unit/test_launchers.py

```diff
@@ -105,10 +106,22 @@ def test_launcher_deleted_remove_tempdir(self):
 
 
 class TestFennecLauncher(unittest.TestCase):
+
+    def setUp(self):
+        self.test_root = '/sdcard/tmp'
+        self.profile = Profile()
+        self.profile_name = os.path.basename(self.profile.profile)
+        # No join because there is a path check later on
```

Comment:
On remote devices, the separator is always '/' - so the line after is normal. Can you remove or update the comment ?